evalml/automl/automl_search.py

```diff
@@ -591,7 +596,7 @@ def _add_baseline_pipelines(self, X, y):
                 desc = desc.ljust(self._MAX_NAME_LEN)
 
                 update_pipeline(logger, desc, len(self._results['pipeline_results']) + 1, self.max_iterations,
-                                self._start)
+                                self._start, '1')
```

Comment:
Why is `'1'` passed in as a string instead of an int?

Reply:
Hmm yeah I think its fine to pass as an int or a string, right? Its the current batch number, going into a format string in `update_pipeline`

Actually, @christopherbunn, can we please use `self._automl_algorithm.batch_number` here instead? The automl algorithm is responsible for keeping track of the batch number. In this case it'll [start at 0](https://github.com/alteryx/evalml/blob/main/evalml/automl/automl_algorithm/automl_algorithm.py#L38)